src/include/miopen/solver.hpp

```diff
@@ -2345,26 +2417,53 @@ struct PerformanceConfigConvOclBwdWrw2
     std::string ToString() const;
 };
 
+// Unable to use ConvTunableSolver as a base class
+// because PerformanceConfigConvOclBwdWrw2 is a class template
 template <int N_BATCH_LOOPS>
-struct ConvOclBwdWrW2 : ConvSolver
+struct ConvOclBwdWrW2 : ConvTunableSolverBase
```

Comment:
Is it possible to inherit from `ConvTunableSolver<PerformanceConfigConvOclBwdWrw2<N_BATCH_LOOPS>>`?

Reply:
Yes, I sort of figured it out.